libs/shared/components/src/lib/button/button.tsx

```diff
@@ -6,6 +6,8 @@ export interface ButtonProps {
   disabled?: boolean;
   size?: 'lg' | 'md' | 'sm';
   color?: string;
+  center?: boolean;
+  marginTop?: string;
```

Comment:
Rather than adding all styles one by one, add a `styles` property which is then mapped to the styles prop of the button,